Lombiq.Tests.UI/Extensions/OrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -0,0 +1,199 @@
+                    .Get(By.CssSelector($"a[href=\"/media/{documentName}\"]"))
+                    .ClickReliablyAsync(context);
+                context.SwitchToFirstWindow();
+
+                context.WaitForPageLoad();
+                await context.GoToAdminRelativeUrlAsync(mediaPath);
+
+                await context
+                    .Get(By.CssSelector("#folder-tree .treeroot .folder-actions")) // #spell-check-ignore-line
+                    .ClickReliablyAsync(context);
+
+                context.Get(By.Id("create-folder-name")).SendKeys("Example Folder");
+
+                await context.ClickReliablyOnAsync(By.Id("modalFooterOk"));
+
+                // Wait until new folder is created.
+                context.Exists(
+                    By.XPath("//div[contains(@class, 'alert-info') and contains(.,'This folder is empty')]"));
+
+                context.UploadSamplePngByIdOfAnyVisibility("fileupload"); // #spell-check-ignore-line
+                context.UploadSamplePdfByIdOfAnyVisibility("fileupload"); // #spell-check-ignore-line
+                context.WaitForPageLoad();
+
+                var image = context.Get(By.XPath($"//span[contains(text(), '{imageName}')]"));
+
+                context.Exists(By.XPath($"//span[contains(text(), '{documentName}')]"));
+
+                await image.ClickReliablyAsync(context);
+
+                await context
+                    .Get(By.XPath($"//span[contains(text(), '{imageName}')]/ancestor::tr"))
+                    .Get(By.CssSelector("a.btn.btn-link.btn-sm.delete-button"))
+                    .ClickReliablyAsync(context);
+
+                await context.ClickModalOkAsync();
+                context.WaitForPageLoad();
+                await context.GoToAdminRelativeUrlAsync(mediaPath);
+
+                context.Missing(By.XPath("//span[text()=' Image.png ' and @class='break-word']"));
+
+                var deleteFolderButton =
+                    context.Get(By.CssSelector("#folder-tree  li.selected  div.btn-group.folder-actions .svg-inline--fa.fa-trash"));
+                await deleteFolderButton.ClickReliablyAsync(context);
+
+                await context.ClickModalOkAsync();
+                context.WaitForPageLoad();
+                await context.GoToAdminRelativeUrlAsync(mediaPath);
+
+                context.Missing(By.XPath("//div[text()='Example Folder' and @class='folder-name ms-2']"));
+            });
+
+    public static Task TestAuditTrailAsync(this UITestContext context) =>
+    context.ExecuteTestAsync(
+        "Test Audit Trail",
+        async () =>
+        {
+            var auditTrailPath = "/AuditTrail";
+
+            await context.EnableFeatureDirectlyAsync("OrchardCore.AuditTrail");
+            await context.GoToAdminRelativeUrlAsync("/Settings" + auditTrailPath);
+
+            await context.GoToEditorTabAsync("Content");
+
+            await context.SetCheckboxValueAsync(By.XPath("//input[@value='Page']"), isChecked: true);
```

Comment:
`SetCheckboxValueAsync()` always has `isChecked: true` in the codebase, so add that as the default for the parameter instead.